axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -131,7 +131,7 @@
     /**
      * Interval (in ms) for keep alive requests, 0 is keep-alive disabled
```

Comment:
Please indicate the default in Javadoc. The application.properties autocomplete support is generated off this class.

Reply:
Done!